src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
```

Comment:
HashUtil should have private constructor, so nobody could create instances of it